### ISSUE TITLE
a11y: Color Contrast - Add Extension Repositories

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13713]
Insufficient color contrast for standard text

## Actual result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground Text Color: #3D98D3
Background Text Color: #FFFFFF
Ratio: 3.16:1 
Main Content
- Cancel button (on default state) 

Foreground Text Color: #FFFFFF
Background Text Color: #3D98D3
Ratio: 3.16:1 
Main Content
- Add button (on default state

Foreground Text Color:  #FFFFFF 
Background Text Color: #297DB4
Ratio: 4.48:1 
Main Content
- Cancel and Add Button (on focus) "

## Expected result

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Fails VPAT:

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
> 
> Instances:
> Foreground Text Color: #2F68E3
> Background Text Color: #EDF2FC
> Ratio: 4.4:1
> Main Content
> - Cancel Button (on focus and hover)  
> 
> Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text."

Issue 222
